package.json

```diff
@@ -52,6 +52,7 @@
     "sinon": "=2.3.1"
   },
   "dependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
I can recommend the following to prevent these issues
```
npm config set save-exact true
```
https://stackoverflow.com/questions/30656189/make-npm-install-save-add-a-strict-version-to-package-json